src/utils/builder.ts

```diff
@@ -381,23 +382,20 @@ export function setTransitionCondition(
   startNodeKey: string,
   transitionId: string,
   description: string,
-  elm: ElmLibrary,
-  criteriaSource?: string,
-  cql?: string
+  criteria: Criteria
 ): Pathway {
   return produce(pathway, (draftPathway: Pathway) => {
     const foundTransition = draftPathway.nodes[startNodeKey]?.transitions?.find(
       (transition: Transition) => transition.id === transitionId
     );
 
-    if (foundTransition) {
+    if (foundTransition)
       foundTransition.condition = {
-        description,
-        elm,
-        criteriaSource: criteriaSource ?? getElmStatement(elm).name,
-        cql: cql ?? ''
+        description: description,
+        cql: criteria.statement,
```

Comment:
When I export the pathway it looks like the cql generated by the criteria builder is no longer included in the criteria. I'm assuming it has to do with some of the changes in this function. Did we still want to include the raw cql in the pathway?

Reply:
From previous discussions we decided the cql should just be the statement name and the raw library exported separately (which right now is just in ELM). @dehall is this still the plan with current developments?

Reply:
So there are 2 options - include the CQL statement directly in this field, or include a name for the statement in this field and the CQL itself in an exported library. There are probably advantages to both, but at this point I don't have any strong preference as long as the statement goes somewhere. We do want to include the CQL as well as ELM because cqf-ruler can only execute CQL, not ELM.

Reply:
Ok I just wanted to make sure this wasn't a rebase error. I think the way Ben has it now should be fine then.

Reply:
Yeah so we will eventually need something to export the CQL in the CPG, but that's not part of this task. Right now it's exported as ELM which works but cqf-ruler can't parse that